pkg/crc/machine/start.go

```diff
@@ -137,19 +132,27 @@ func growRootFileSystem(sshRunner *crcssh.Runner, preset crcPreset.Preset) error
 	if _, _, err := sshRunner.RunPrivileged("Remounting /sysroot read/write", "mount -o remount,rw /sysroot"); err != nil {
```

Comment:
you also use `rootMount` here, which justifies re-use for clarity

Reply:
updated with `rootFS` variable